### DISCUSSION TITLE
FAQ for the miniDexed WIKI

### DISCUSSION THREAD

**Discussion by Banana71** [probonopd/MiniDexed#642] [Ideas]
Since some questions are repeated here, I'm currently working on an FAQ section in the miniDexed-WIKI. The first draft is in German and is still in the soundplantage 😃 
https://github.com/Banana71/Soundplantage/wiki/FAQ
Do you have any ideas about which questions should be included?

**Comment by probonopd** [comment]
Well, the classic would be "my (...) is not working". But isn't https://github.com/probonopd/MiniDexed/discussions kinda the knowledge base for such questions and their answers?
